mlx_graphs/datasets/planetoid.py

```diff
@@ -0,0 +1,201 @@
+import os.path as osp
+import warnings
+from itertools import repeat
+from typing import List, Literal, Optional
+
+import fsspec
```

Comment:
this needs to be added to the dependencies if we really need it - I feel `utils/fs.py` can also be implemented without it.